tasks/section5.yml

```diff
@@ -211,15 +321,19 @@
       - rule_5.2.3
 
 - name: "SCORED | 5.2.3 | PATCH | Ensure SSH LogLevel is set to INFO"
-  command: /bin/true
+  lineinfile:
+      state: present
+      dest: /etc/ssh/sshd_config
+      regexp: '^(#)?LogLevel INFO\d'
+      line: 'LogLevel INFO'
   tags:
       - level1
       - level2
       - patch
       - rule_5.2.3
 
 - name: "SCORED | 5.2.4 | AUDIT | Ensure SSH X11 forwarding is disabled"
-  command: /bin/true
+  command: grep "^X11Forwarding no" /etc/ssh/sshd_config
   register: result
```

Comment:
Audit tasks should register verbose variable name that ends with `_audit`. As you are adding AUDIT checks please change `register: result` to `register: `*`descriptive_check_name_audit`*